docs/Limitations.md

```diff
@@ -0,0 +1,19 @@
+# Limitations of Azure DevOps Demo Generator
+
+Azure DevOps Demo Generator is to create a demo set up with predefined templates, it has some limitations, which are listed below. This can't be treated as migration or copying project tool from one Organization to another.
+
+## Following are the limitations of Azure DevOps Demo Generator and Template Builder
+
+|||
+|------|-----|
+|Custom process template|This is supported to an extent, but not all the options available as compared to standard process template. Organization must contains same process template Id|
```

Comment:
I want to make sure I understand...  this is "Fully supported except that the exact same custom template must already exist in the target organization."  Is that correct?  Are there still limitations even if that is true? And if that isn't true, does it still provide a partial migration or is it a complete fail?

Reply:
Though having same process template there will be some limitations with setting up board colums. Rest other things like work items, pipelines, repose will work as is.